Packs/Base/Scripts/SaneDocReportV2/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- Updating sane doc reports and adding customer logo. 
```

Comment:
Please use the new release notes format
demisto-sdk update-release-notes

Reply:
```
Starting to update release notes.
No changes were detected.
```
?

Reply:
@amshamah419 can you take a look at this?

Reply:
@agamm Can you try updating your `demisto-sdk`?

Reply:
@ronykoz @amshamah419  I updated before:
![image](https://user-images.githubusercontent.com/1269911/85228935-50a88b00-b3ef-11ea-9764-d94cda8443de.png)

(Already tried: `pip3 install --upgrade demisto-sdk`) anything I need to change in the code / make changes ?